scripts/utilities/cellspace.py

```diff
@@ -345,9 +413,19 @@ def write(self, filename=None, temp_mod=False):
                 cells = self.temp_mod_cells.astype(self.cell_data_type)
             else:
                 cells = self.cells.astype(self.cell_data_type)
-        # write the cells
-        cells.tofile(f)
-        f.close()
+            
+        if compressed:
+            # if no .gz at end of file, add it
+            if not filename.endswith('.gz'):
+                filename = filname + '.gz'
```

Comment:
Spelling mistake `filname`?

Reply:
I think I never tested it to get that line to execute. Changed to filename

Reply:
Thanks.